tests/puppeteer/campaigns/utils/helpers.js

```diff
@@ -3,8 +3,23 @@ require('./globals');
 const puppeteer = require('puppeteer');
 
 module.exports = {
-  async createBrowser() {
-    return puppeteer.launch(global.BROWSER_CONFIG);
+  /**
+   * Create puppeteer browser
+   * @param attempt, number of attempts to restart browser creation if function throw error
+   * @return {Promise<browser>}
+   */
+  async createBrowser(attempt = 1) {
+    try {
+      const browser = await puppeteer.launch(global.BROWSER_CONFIG);
+      return browser;
```

Comment:
```suggestion
      return await puppeteer.launch(global.BROWSER_CONFIG);
```

Why not this ?

Reply:
Is there any advantage at doing this except not using a variable ?

Reply:
The var is useless there, so yes

Reply:
saw with @NeOMakinG and after checking [this](https://jakearchibald.com/2017/await-vs-return-vs-return-await/),
I can use `return await puppeteer.launch(global.BROWSER_CONFIG);` 
but not `return puppeteer.launch(global.BROWSER_CONFIG);`

Reply:
Fixed, thanks all